python/tests/test_JMXQuery.py

```diff
@@ -126,8 +126,16 @@ def test_threading():
 def printMetrics(metrics):
     for metric in metrics:
         if metric.metric_name:
-            print(f"{metric.metric_name}<{metric.metric_labels}> == {metric.value}")
+            print("{%s}<{%s}> == {%s}" % (
+                metric.metric_name,
+                metric.metric_labels,
+                metric.value
+            ))
```

Comment:
For this `print()` code to work as expected, this line
```
from __future__ import print_function
```
should be added to the top of the module.  It should be the first line of code, but comments may appear above it.

Reply:
Implemented per your recommendation.